src/app/blog/[slug]/not-found.tsx

```
@@ -1,173 +1,24 @@
-'use client';
+"use client";
 
-import { Box, Container, Typography, Button, useTheme, Paper } from '@mui/material';
-import Link from 'next/link';
-import { FileX } from 'lucide-react';
-import { motion } from 'framer-motion';
+import {
+  useTheme,
```

ESLint

[error] 'useTheme' is defined but never used.

(@typescript-eslint/no-unused-vars)